src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -398,6 +393,94 @@ public void GetIssues_UnboundSolution_NoErrors()
             matches.Should().BeEmpty();
         }
 
+        [TestMethod]
+        public void GetIssues_ErrorInInitialFetchTask_IsSuppressed()
```

Comment:
New tests to check what happens if an exception is thrown on a background thread during the initial fetch, or when a re-fetch is triggered by either the solution binding changing or the timer elapsing.